titanic.ipynb

```diff
@@ -8,6 +8,7 @@
         "## Titanic 3\n",
```

Comment:
_Commented on notebook `titanic.ipynb` Cell 1 Line 1_
```markdown
## Titanic 3
```
A markdown comment to amrk as resolved

_Respond and view the context [here](http://localhost:3000/smith-kyle/notebook-diff-tests/pull/10)._

Reply:
here's a reply